server/base/src/main/java/org/apache/accumulo/server/rpc/TServerUtils.java

```diff
@@ -591,34 +590,19 @@ public static ServerAddress startTServer(final AccumuloConfiguration conf,
     }
   }
 
-  /**
-   * @see #startTServer(ThriftServerType, TimedProcessor, TProtocolFactory, String, String, int,
-   *      long, AccumuloConfiguration, long, long, SslConnectionParams, SaslServerConnectionParams,
-   *      long, HostAndPort...)
-   */
-  public static ServerAddress startTServer(ThriftServerType serverType, TimedProcessor processor,
-      String serverName, String threadName, int numThreads, long threadTimeOut,
-      final AccumuloConfiguration conf, long timeBetweenThreadChecks, long maxMessageSize,
-      SslConnectionParams sslParams, SaslServerConnectionParams saslParams,
-      long serverSocketTimeout, HostAndPort... addresses) throws TTransportException {
-    return startTServer(serverType, processor, ThriftUtil.protocolFactory(), serverName, threadName,
-        numThreads, threadTimeOut, conf, timeBetweenThreadChecks, maxMessageSize, sslParams,
-        saslParams, serverSocketTimeout, addresses);
-  }
-
   /**
    * Start the appropriate Thrift server (SSL or non-blocking server) for the given parameters.
    * Non-null SSL parameters will cause an SSL server to be started.
    *
    * @return A ServerAddress encapsulating the Thrift server created and the host/port which it is
    *         bound to.
    */
-  public static ServerAddress startTServer(ThriftServerType serverType, TimedProcessor processor,
-      TProtocolFactory protocolFactory, String serverName, String threadName, int numThreads,
-      long threadTimeOut, final AccumuloConfiguration conf, long timeBetweenThreadChecks,
-      long maxMessageSize, SslConnectionParams sslParams, SaslServerConnectionParams saslParams,
```

Comment:
It looks like this is currently being used in accumulo-proxy, [here](https://github.com/apache/accumulo-proxy/blob/95f9c908bc086a049c322191ec9a8e767f886773/src/main/java/org/apache/accumulo/proxy/Proxy.java#L270). Not sure if that should influence whether or not this is kept but thought I should point it out.

Reply:
It's not public API. I wonder if we can avoid using it in the proxy.